lisa/sut_orchestrator/azure/features.py

```diff
@@ -668,6 +669,67 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
             len(all_nics) - self.origin_extra_synthetic_nics_count - 1
         )
 
+    def create_route_table(
```

Comment:
Where does it suppose to use? Should it be defined in features as a standard method for all platforms, or only for Azure?

Reply:
It's going to be used in the l3fwd test specifically. I don't have plans to support AWS with the dpdk tests

Reply:
It's not only AWS, if LISA supports HyperV, it may need to be covered too.

Reply:
true

Reply:
Added the prototypes